cloudbuild.yaml

```diff
@@ -74,6 +74,10 @@ steps:
         ./gradlew -PdisablePreDex checkCode 2> check-logs.txt || echo "fail" > build-status.txt
         cat check-logs.txt
 
+        if [[ "${_PUSH_TO_MASTER}" ]]; then
```

Comment:
Could you please add a comment explaining when _PUSH_TO_MASTER is true? Is it run before GitHub merges an approved PR with master? Will updates block merge?

Reply:
Added notes. No, it won't block merges.